apps/ledger-live-desktop/tests/page/swap.page.ts

```diff
@@ -123,6 +132,13 @@ export class SwapPage extends AppPage {
     return waitFor(() => this.exchangeButton.isEnabled(), 250, 10000);
   }
 
+  @step("Click Exchange button")
+  async clickExchangeButton(electronApp: ElectronApplication, provider: string) {
+    await this.page.waitForLoadState("domcontentloaded");
```

Comment:
It'll use one line instead of using one line, not sure of the benefit but yes it's doable

Reply:
it's mostly about string that is used here, I would prefer it having in one place in case we need to do some changes so it would be done only in one place